src/script/conversation/ClientMismatchHandler.ts

```diff
@@ -191,24 +183,19 @@ export class ClientMismatchHandler {
   async removePayload(
     recipients: UserClients,
     clientFn: (userId: string, clientId: string) => void | Promise<void>,
-    conversationEntity: Conversation = undefined,
     payload: NewOTRMessage,
+    conversationEntity?: Conversation,
```

Comment:
Why has this become optional?

```suggestion
    conversationEntity: Conversation,
```

Reply:
It was already optional, it was just not following the standards because the parameter after this one was not optional.